opal/mca/common/ucx/common_ucx_wpool.c

```diff
@@ -67,9 +69,9 @@ _winfo_create(opal_common_ucx_wpool_t *wpool)
 
     OBJ_CONSTRUCT(&winfo->mutex, opal_recursive_mutex_t);
     winfo->worker = worker;
-    winfo->endpoints = NULL;
-    winfo->comm_size = 0;
-    winfo->inflight_ops = NULL;
+    OBJ_CONSTRUCT(&winfo->ep_iops, opal_hash_table_t);
+    opal_hash_table_init(&winfo->ep_iops, 256);
```

Comment:
Can we limit the size of the hash table for communicators smaller than 256 ranks?